SpiffWorkflow/bpmn/specs/data_spec.py

```diff
@@ -24,6 +24,33 @@ def set(self, my_task, **kwargs):
         raise NotImplementedError
 
 
+class BpmnDataStoreSpecification:
```

Comment:
I think this should inherit from `BpmnDataSpecification`.  Also, that class no longer has a copy method so not sure there's a reason for it to exist here (not objecting to having it, but don't know if there's a reason beyond the fact the data spec used to have it).

Reply:
Must have just been a hold over from when I started the branch, will update.